src/App.tsx

```diff
@@ -1,24 +1,234 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+// import React, { useEffect, useState } from 'react';
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)